frontend/src/scenes/trends/viz/ActionsPie.tsx

```diff
@@ -46,41 +48,44 @@ export function ActionsPie({ inSharedMode, showPersonsModal = true }: ChartParam
+                <div className="actions-pie-component">
+                    <div className="pie-chart">
+                        <PieChart
+                            data-attr="trend-pie-graph"
+                            hiddenLegendKeys={hiddenLegendKeys}
+                            type={GraphType.Pie}
+                            datasets={data}
+                            labels={data[0].labels}
+                            labelGroupType={labelGroupType}
+                            inSharedMode={!!inSharedMode}
+                            showPersonsModal={showPersonsModal}
+                            filters={insight.filters}
+                            onClick={
+                                !showPersonsModal || insight.filters?.formula
+                                    ? undefined
+                                    : (payload) => {
+                                          const { points, index, crossDataset } = payload
+                                          const dataset = points.referencePoint.dataset
+                                          const label = dataset.labels?.[index]
 
-                                      const urls = urlsForDatasets(crossDataset, index)
-                                      const selectedUrl = urls[index]?.value
+                                          const urls = urlsForDatasets(crossDataset, index)
+                                          const selectedUrl = urls[index]?.value
 
-                                      if (selectedUrl) {
-                                          openPersonsModal({
-                                              urls,
-                                              urlsIndex: index,
-                                              title: <PropertyKeyInfo value={label || ''} disablePopover />,
-                                          })
+                                          if (selectedUrl) {
+                                              openPersonsModal({
+                                                  urls,
+                                                  urlsIndex: index,
+                                                  title: <PropertyKeyInfo value={label || ''} disablePopover />,
+                                              })
+                                          }
                                       }
-                                  }
-                        }
-                    />
+                            }
+                        />
+                    </div>
+                    <h1 className="text-7xl text-center">{formatAggregationAxisValue(insight.filters, total)}</h1>
                 </div>
-                <h1 className="text-7xl text-center">{formatAggregationAxisValue(insight.filters, total)}</h1>
+                {inCardView && filters.show_legend && <InsightLegend inCardView className={'mr-4'} />}
```

Comment:
This className could be removed in favor of a "gap-4" on the parent className as it is flexed (which also means you don't need to exposes className as a prop anymore for InsightLegend)

Reply:
`gap-4` still left the legend up against the edge of the container. But I could pad the container instead of margin-ing the child 👨‍🍳 👌